app/src/main/cpp/skyline/input/npad_device.cpp

```diff
@@ -260,10 +260,10 @@ namespace skyline::input {
         if (!connectionState.connected)
             return;
 
-        auto &controllerEntry = GetNextEntry(*controllerInfo);
-        auto &defaultEntry = GetNextEntry(section.defaultController);
+        auto &controllerEntry{GetNextEntry(*controllerInfo)};
+        auto &defaultEntry{GetNextEntry(section.defaultController)};
 
-        constexpr i16 threshold = std::numeric_limits<i16>::max() / 2; // A 50% deadzone for the stick buttons
+        constexpr i16 threshold{std::numeric_limits<i16>::max() / 2}; // A 50% deadzone for the stick buttons
```

Comment:
auto can be used here

Reply:
`i16` is kind of smaller than `auto`, we're somewhat inconsistent with if we use integer types or auto for return types from them but I'm fine with either. I don't think it's something that should be remedied.